src/gameObjects/player.js

```diff
@@ -1,7 +1,10 @@
 export default class Player {
   constructor(scene, x, y) {
     this.scene = scene;
+    this.playerSpeed = 2;
```

Comment:
**suggestion**: Consider making `playerSpeed` a configurable parameter to allow for easier adjustments and testing of different speeds.